bin/argos

```diff
@@ -1,2 +1,4 @@
 #!/usr/bin/env node
+
+process.env.NODE_ENV = 'production'
```

Comment:
@oliviertassinari use [debug](https://github.com/visionmedia/debug) if you want additional log, it can be useful for end user to activate it.